web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
Ohhhh interesting thanks for explaining. Didn't realize that the apps were considered a level. I was honestly confused as to where the fourth level came in 😆